.travis.yml

```diff
@@ -4,6 +4,7 @@ services:
 language: bash
 script:
   - make shellcheck
+  - make SWARM_VERSION=1.2.2 CONSUL_VERSION=0.6.4 get-cluster-lab-images
```

Comment:
I would like to remove this extra step. This should all be done with a simple "make sd-card". This should be possible to put the target as dependency to the Makefile:

```
sd-image: get-cluster-lab-images build
```
 
We also should put the versions out of the .travis.yml into another place.